djstripe/migrations/0013_alter_payout_destination.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 3.2.12 on 2022-03-29 06:49
+
+from django.db import migrations
+
+from sql.utils import get_sql_for_connection
+
+
+def forwards_func(apps, schema_editor):
+    vendor, sql_statement = get_sql_for_connection(schema_editor, "forward")
+    with schema_editor.connection.cursor() as cursor:
+        if vendor == "sqlite":
+            cursor.executescript(sql_statement)
+        else:
+            cursor.execute(sql_statement)
+
+
+def reverse_func(apps, schema_editor):
+    vendor, sql_statement = get_sql_for_connection(schema_editor, "backward")
+    with schema_editor.connection.cursor() as cursor:
+        if vendor == "sqlite":
+            cursor.executescript(sql_statement)
```

Comment:
Use `schema_editor.execute`. cf https://stackoverflow.com/questions/32384547/django-migration-runsql-conditional-on-database-type

Reply:
That is not [recommended](https://docs.djangoproject.com/en/4.1/ref/migration-operations/#runpython):

> By default, RunPython will run its contents inside a transaction on databases that do not support DDL transactions (for example, MySQL and Oracle). This should be safe, but may cause a crash if you attempt to use the schema_editor provided on these backends; in this case, pass atomic=False to the RunPython operation.

Would need to tweak the `atomic` kwarg as well. 

Django docs themselves [recommend](https://docs.djangoproject.com/en/4.1/topics/db/sql/#executing-custom-sql-directly) using `cursor` to execute custom SQL queries directly.